### ISSUE TITLE
Parse error inside component fragments.

### ISSUE THREAD

**Issue by richburdon** [jimkyndemeyer/js-graphql-intellij-plugin#36]
Fragment references that pass in variables aren't parsed correctly.

The following is correct syntax (compiles and runs), but generates an error:

```
  fragments: {
    viewer: (variables) => Relay.QL`
      fragment on Viewer {
        ${Picker.getFragment('viewer', { filter: variables.filter })} <== PARSE ERROR HERE
      }
    `,
  }
```

ERROR: Syntax Error GraphQL (100:68) Expected Name, found )

(Also the error isn't well-formed).

**Comment by jimkyndemeyer** [comment]
Hi Rich.

Thanks for catching this.

I was able to reproduce. What do you mean by "(Also the error isn't well-formed)."?

**Comment by jimkyndemeyer** [comment]
Fixed and released in v1.4.3.
